specification/desktopvirtualization/resource-manager/Microsoft.DesktopVirtualization/preview/2024-04-08-preview/desktopvirtualization.json

```diff
@@ -7171,6 +7213,41 @@
         }
       }
     },
+    "RegistrationInfoList": {
+      "type": "object",
+      "title": "RegistrationInfoList",
+      "description": "List of RegistrationInfo definitions.",
+      "properties": {
+        "value": {
+          "type": "array",
```

Comment:
>type [](http://example.com/codeflow?start=11&length=4)

an x-ms-identifer is needed here. to indicate identifying properties in the array. [https://github.com/Azure/autorest/tree/main/docs/extensions#x-ms-identifiers](https://github.com/Azure/autorest/tree/main/docs/extensions#x-ms-identifiers)